src/javascript/_common/base/currency_base.js

```diff
@@ -4,7 +4,7 @@ const getPropertyValue = require('../utility').getPropertyValue;
 
 let currencies_config = {};
 
-const getTextFormat = (number, currency) => `${currency} ${addComma(number, getDecimalPlaces(currency), isCryptocurrency(currency))}`;
+const getTextFormat = (number, currency) => `${currency} ${addComma(number, getDecimalPlaces(currency), isCryptocurrency(currency) && currency !== 'UST')}`;
```

Comment:
If 2 decimal places is true for stable coins, maybe we should move this check inside `getDecimalPlaces` so other users of this function may benefit as well.

Reply:
we shouldn't hardcode this, if it's based on decimal places for that crypto currency we should check that from the currency config object